CODEOWNERS

```diff
@@ -14,80 +14,78 @@
 /doc/gov/chapters/chapter04.md       @jbeltranatl
 /doc/gov/chapters/chapter05.md       @jbeltranatl
 /doc/gov/chapters/chapter06.md       @jbeltranatl
-
-# Reference Model Core WS
-/doc/ref_model/chapters/chapter01.md    @rabiabdel @markshostak
-/doc/ref_model/chapters/chapter02.md    @rabiabdel @markshostak
-/doc/ref_model/chapters/chapter03.md    @rabiabdel @markshostak
-/doc/ref_model/chapters/chapter04.md    @rabiabdel @markshostak
-/doc/ref_model/chapters/chapter05.md    @rabiabdel @markshostak
-/doc/ref_model/chapters/appendix-a.md    @rabiabdel @markshostak
-
-# Reference Model OPS WS
-/doc/ref_model/chapters/chapter06.md    @rabiabdel @ASawwaf
-/doc/ref_model/chapters/chapter07.md    @rabiabdel @ASawwaf
-/doc/ref_model/chapters/chapter09.md    @rabiabdel @ASawwaf
-
-# Reference Model COM WS
-/doc/ref_model/chapters/chapter08.md        @rabiabdel @Gaoweitao 
-/doc/ref_model/chapters/chapter08-annex.md  @rabiabdel @Gaoweitao 
-
-# Reference Architectures 1 Core
+/doc/gov/chapters/chapter07.md       @jbeltranatl
+/doc/gov/chapters/chapter08.md       @jbeltranatl
+/doc/gov/chapters/chapter09.md       @jbeltranatl
+/doc/gov/chapters/chapter10.md       @jbeltranatl
+/doc/gov/chapters/chapter11.md       @jbeltranatl
+
+# Reference Model
+/doc/ref_model/chapters/chapter01.md        @rabiabdel @kedmison
+/doc/ref_model/chapters/chapter02.md        @rabiabdel @kedmison
+/doc/ref_model/chapters/chapter03.md        @rabiabdel @kedmison
+/doc/ref_model/chapters/chapter04.md        @rabiabdel @kedmison
+/doc/ref_model/chapters/chapter05.md        @rabiabdel @kedmison
+/doc/ref_model/chapters/chapter06.md        @rabiabdel @kedmison
+/doc/ref_model/chapters/chapter07.md        @rabiabdel @kedmison
+/doc/ref_model/chapters/chapter09.md        @rabiabdel @kedmison
+/doc/ref_model/chapters/chapter08.md        @rabiabdel @kedmison 
+/doc/ref_model/chapters/appendix-a.md       @rabiabdel @kedmison
+/doc/ref_model/chapters/chapter08-annex.md  @rabiabdel @kedmison 
+
+# Reference Architectures 1
 /doc/ref_arch/openstack/chapters/chapter01.md        @rabiabdel @pgoyal1
 /doc/ref_arch/openstack/chapters/chapter02.md        @rabiabdel @pgoyal1 
 /doc/ref_arch/openstack/chapters/chapter03.md        @rabiabdel @pgoyal1 
 /doc/ref_arch/openstack/chapters/chapter04.md        @rabiabdel @pgoyal1 
+/doc/ref_arch/openstack/chapters/chapter05.md        @rabiabdel @pgoyal1
+/doc/ref_arch/openstack/chapters/chapter06.md        @rabiabdel @pgoyal1 
+/doc/ref_arch/openstack/chapters/chapter07.md        @rabiabdel @pgoyal1 
+/doc/ref_arch/openstack/chapters/chapter01.md        @rabiabdel @pgoyal1
```

Comment:
Typo: This should be RA-1 chapter08 not Chapter01

Reply:
fixed herre: https://github.com/cntt-n/CNTT/pull/1062/commits/3f9b62c74553f5093528cac9df375e25b5f98443